src/Microsoft.Extensions.ProjectModel.Sources/MsBuild/MsBuildProjectContext.cs

```diff
@@ -35,7 +35,19 @@ public string FindProperty(string propertyName)
         public string ProjectName => FindProperty("ProjectName") ?? _name;
         public string Configuration { get; }
 
-        public NuGetFramework TargetFramework => NuGetFramework.Parse(FindProperty("NuGetTargetMoniker"));
+        public NuGetFramework TargetFramework
```

Comment:
Would probably be easier to just swap this to `IEnumerable<NuGetFramework> TargetFrameworks`. That way consuming libraries can choose which framework to use.